src/Pages/hth/MainHTH/MainHTH.js

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import "./MainHTH.scss";
+import Article from "./Components/Article/Article";
+import SearchView from "./Components/SearchView/SearchView";
+
+class MainHTH extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      clicked: false,
+      searchText: "",
+      users: [],
+      searchResult: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/hth/data.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          users: res.users,
+        });
+      });
+  }
```

Comment:
```suggestion
  componentDidMount() {
    this.getUserData()
  }
  
  const getUserData = () => {
    fetch("/data/hth/data.json")
      .then((res) => res.json())
      .then(({ users }) => this.setState({ users });
  }
```
- CDM 안에 직접 로직을 작성하기 보다는 의미 있는 함수 단위로 분리하면 좋을 것 같습니다
- get method 생략해줄 수 있습니다
- 함수 파라미터 자리에 구조분해 할당 해줄 수 있습니다
- 객체 단축 속성명 사용할 수 있습니다